### DISCUSSION TITLE
Hubot not responding to "st2 list" command

### DISCUSSION THREAD

**Discussion by mitthat29** [StackStorm/st2#5729] [ChatOps]
<img width="570" alt="Untitled" src="https://user-images.githubusercontent.com/111504301/190378421-99423772-68ab-4fc3-a1d4-1ffedabf131c.png">

Hi, 
I tried running this self-check script for hubot but it didn't make through. I have tried the suggestions given by this error and tried reinstalling chatops. Also verified that my bot name is "hubot" only in the st2chatops.env file. Also, couldn't find any log file for chatops at the path mentioned in the suggestions.

**Comment by rbullers** [comment]
Having the exact same issue on a centos8 server - slack bot will only show help/pack commands 

![image](https://user-images.githubusercontent.com/88460527/228800655-35a81c45-3ff2-483a-be8a-7a1f387361db.png)
